src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/jpa/VisitRestriction.kt

```diff
@@ -0,0 +1,9 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.jpa
+
+@Suppress("unused")
+enum class VisitRestriction(
+  val description: String,
+) {
+  OPEN("Open"),
```

Comment:
I wonder what use the description is?

Reply:
Yep I guess I understand the logging aspect.. I tend to avoid because some times comment is out sync... and enum should be descriptive enough (self documenting and all
)

Reply:
I think we need to talk to the UI team and raise a ticket to remove all of these strings - its come up a number of times now but is being put off until after the MVP release.